inicheck/utilities.py

```diff
@@ -64,16 +64,16 @@ def find_options_in_recipes(recipes, choice_search,
         final_choices:
 
     """
-    decisions = {}
-    conditional = ["any" for i in range(3)]
+    # decisions = {}
+    # conditional = ["any" for i in range(3)]
```

Comment:
Are they unused?

Reply:
Some of these I wanted @micahjohnson150 to make sure they aren't needed before full removal.

Reply:
I am fairly confident with Flake8 detecting those correctly.
Again a vote to add a #TODO marker.

Reply:
Yea I agree. These are unused. Must have been a hold over from pre changelog (based on the doc string which is now out of date)